ibm_catalog.json

```diff
@@ -63,6 +63,174 @@
+                "options": [
+                  {
+                    "displayname": "public",
+                    "value": "public"
+                  },
+                  {
+                    "displayname": "private",
+                    "value": "private"
+                  },
+                  {
+                    "displayname": "public-and-private",
+                    "value": "public-and-private"
+                  }
+                ]
+              },
+              {
+                "key": "management_endpoint_type_for_bucket",
+                "required": true,
+                "default_value": "private",
+                "options": [
+                  {
+                    "displayname": "public",
+                    "value": "public"
+                  },
+                  {
+                    "displayname": "private",
+                    "value": "private"
+                  },
+                  {
+                    "displayname": "direct",
+                    "value": "direct"
+                  }
+                ]
+              },
+              {
+                "key": "at_cos_target_bucket_class",
+                "required": true,
+                "default_value": "smart",
+                "options": [
+                  {
+                    "displayname": "standard",
+                    "value": "standard"
+                  },
+                  {
+                    "displayname": "vault",
+                    "value": "vault"
+                  },
+                  {
+                    "displayname": "cold",
+                    "value": "cold"
+                  },
+                  {
+                    "displayname": "smart",
+                    "value": "smart"
+                  },
+                  {
+                    "displayname": "onerate_active",
+                    "value": "onerate_active"
+                  }
+                ]
+              },
+              {
+                "key": "log_archive_cos_bucket_class",
+                "required": true,
+                "default_value": "smart",
+                "options": [
+                  {
+                    "displayname": "standard",
+                    "value": "standard"
+                  },
+                  {
+                    "displayname": "vault",
+                    "value": "vault"
+                  },
+                  {
+                    "displayname": "cold",
+                    "value": "cold"
+                  },
+                  {
+                    "displayname": "smart",
+                    "value": "smart"
+                  },
+                  {
+                    "displayname": "onerate_active",
+                    "value": "onerate_active"
+                  }
+                ]
+              },
+              {
+                "key": "kms_endpoint_type",
+                "required": true,
+                "default_value": "private",
```

Comment:
you can remove `default_value` here so we don't have to maintain it in 2 places. If not specified here, it will pick the default from terraform.